### PR TITLE
Improve styling

### PR DESCRIPTION
**Pull request by mayayunx**
Signed-off-by: Ruiyun Xie <ruiyun.xie@tum.de>

### Summary of changes
- Change multiselect to only one line and adjust styling.

### Context and reason for change
- Current styling of multiselect too prominent.

### How can the changes be tested
- The styling can be tested in the UI.

### CODE REVIEW

File: src/Frontend/Components/Filter/FilterMultiSelect.tsx [hunk 1]
```diff
@@ -28,28 +29,42 @@ const FILTERS = [
...
       backgroundColor: OpossumColors.white,
-      padding: '1px 3px',
+      padding: '1px',
     },
   },
   dropDownSelect: {
-    height: '65px',
+    minHeight: '36px',
+    '& svg': {
+      paddingRight: '6px !important',
```
Comment (benedikt-richter) [inline src/Frontend/Components/Filter/FilterMultiSelect.tsx:45]: we should avoid `!important`

File: src/Frontend/Components/Filter/FilterMultiSelect.tsx [hunk 2]
```diff
@@ -28,28 +29,42 @@ const FILTERS = [
...
     flexWrap: 'wrap',
     gap: 0.5,
   },
+  chip: {
+    maxHeight: '19px',
+    fontSize: '12px',
+  },
+  dropdownStyle: {
+    maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+    left: '7px !important',
```
Comment (benedikt-richter) [inline src/Frontend/Components/Filter/FilterMultiSelect.tsx:59]: we should avoid `!important`

File: src/Frontend/Components/Filter/FilterMultiSelect.tsx [hunk 3]
```diff
@@ -28,28 +29,42 @@ const FILTERS = [
...
+      paddingRight: '6px !important',
+    },
   },
   dropDownBox: {
     display: 'flex',
     flexWrap: 'wrap',
     gap: 0.5,
   },
+  chip: {
+    maxHeight: '19px',
```
Comment (benedikt-richter) [inline src/Frontend/Components/Filter/FilterMultiSelect.tsx:54]: ´maxHeight: 19´
also other places

File: src/Frontend/Components/Filter/FilterMultiSelect.tsx [hunk 4]
```diff
@@ -101,14 +121,7 @@ export function FilterMultiSelect(): ReactElement {
...
         )}
-        MenuProps={{
-          PaperProps: {
-            style: {
-              maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
-              width: 170,
-            },
-          },
-        }}
+        MenuProps={{ classes: { paper: classes.dropdownStyle } }}
```
Comment (benedikt-richter) [inline src/Frontend/Components/Filter/FilterMultiSelect.tsx:124]: nice!

**Review by benedikt-richter — COMMENTED** [review]
Looks good!

just some minor comments

**Review by nicarl — APPROVED** [review]

**Pull request merged**